### PR TITLE
UNI-228 #comment Move the "Metric MORE Submenu" from LeftNav to AboveCharts

### PR DESCRIPTION
**Pull request by brev**
UNI-228 #comment Move the "Metric MORE Submenu" from LeftNav to AboveCharts.

Also: Bump packages, Sync license snippets and remove NuPIC reference, etc.

As per recent spec from @marionleborgne.

Please review @lscheinkman or @marionleborgne, thanks

screen shot 2015-11-30 at 3 04 54 pm

### CODE REVIEW

**Comment by rhyolight** [comment]
By analyzing the blame information on this pull request, we identified @lscheinkman and @marionleborgne to be potential reviewers

File: unicorn/package.json [hunk 1]
```diff
@@ -104,15 +104,15 @@
     "babel-core": "5.8.23",
     "babel-eslint": "4.1.6",
     "babel-loader": "5.3.2",
-    "babel-plugin-typecheck": "3.1.0",
+    "babel-plugin-typecheck": "2.0.1",
```
Comment (lscheinkman) [inline unicorn/package.json]: Did you mean to downgrade this package?

File: unicorn/gulpfile.babel.js [hunk 2]
```diff
@@ -63,6 +63,10 @@ gulp.task('webpack', ()  => {
             test: /\.css$/,
             loaders: ['style', 'css']
           },
+          {
+            test: /\.scss$/,
+            loaders: ['style', 'css', 'sass']
```
Comment (lscheinkman) [inline unicorn/gulpfile.babel.js]: Are we using `sass` in this project?

File: unicorn/package.json [hunk 3]
```diff
@@ -120,7 +120,9 @@
     "lev": "3.2.3",
     "mocha": "2.3.4",
     "node-inspector": "0.12.3",
-    "source-map-support": "0.3.3",
+    "node-sass": "3.4.2",
+    "sass-loader": "3.1.2",
```
Comment (lscheinkman) [inline unicorn/package.json]: Are we using `sass` in this project?

**Comment by lscheinkman** [comment]
File Menu icons were truncated. 
screen shot 2015-12-01 at 9 08 57 am

**Comment by lscheinkman** [comment]
After Clicking "STOP" all other buttons disappear.
screen shot 2015-12-01 at 9 13 58 am

**Comment by lscheinkman** [comment]
Missing __Create Model__ button.
The checkbox should only show/hide the model. The active buttons should be based on whether the metric has a model or not. See wireframe (pages 8 and 9) attached to the UNI-228

File: unicorn/js/browser/actions/CreateModel.js [hunk 4]
```diff
@@ -1,19 +1,17 @@
-// Numenta Platform for Intelligent Computing (NuPIC)
```
Comment (lscheinkman) [inline unicorn/js/browser/actions/CreateModel.js]: Good catch! I will update my template.

**Comment by brev** [comment]
@lscheinkman this was just an incremental change not a full feature set

**Comment by brev** [comment]
@lscheinkman nevermind this was just an incremental change not the full UNI-228 + full spec changes

**Pull request closed without merging**